src/LinkDotNet.NCronJob/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.Extensions.DependencyInjection;
+
+namespace LinkDotNet.NCronJob;
+
+/// <summary>
+/// Extensions for the <see cref="IServiceCollection"/> to add cron jobs.
+/// </summary>
+public static partial class ServiceCollectionExtensions
+{
+    /// <summary>
+    /// Adds a job using an anonymous delegate to the service collection that gets executed based on the given cron expression.
+    /// </summary>
+    /// <param name="services">The service collection used to register the services.</param>
+    /// <param name="jobDelegate">The delegate that represents the job to be executed.</param>
```

Comment:
Could start - for sure I would add that the delegate has to have a certain structure (only `void` or `Task`) maybe enriched with a small example that showcases that.